### ISSUE TITLE
line numbering in a fragment

### ISSUE THREAD

**Issue by arlogriffiths** [erc-dharma/tfb-bengalcharters-epigraphy#5]
I have just revised Amandine's original encoding of this item https://dharmalekha.info/texts/INSBengalCharters00050.

Insodoing, I have implemented the line numbering scheme that I am used to for Indonesian copper plates, and that I also used (implicitly) in my 2018 article.

@ryosukefurui : please tell me if, as I suspect, this creates an inconsistency with your editions. If so, I can change and implement continuous numbering from recto to verso.

@danbalogh : let me know if you have any advice on the matter. By the way, @amandinebricout encoded the `<pb>`s outside of the paragraphs. I think I'd normally encode them inside the `<p>`s, right before its first `<lb>`. Both options are allowed by our scheme. Which one is best?

@amandinebricout : I am a bit surprised by the sentence "Re-edited here with small improvements based on direct inspection of the fragment." Because I don't remember ever having seen this fragment with my own eyes. Have you?

**Comment by amandinebricout** [comment]
@arlogriffiths 
I have just checked my notes on my past encodings and I'm really sorry: this sentence was copied and pasted from another encoding (INSBenglaCharters00051). For this one I had added a comment that you need to check this sentence but I forgot to add this comment for this 00050. You can delete this part.

**Comment by arlogriffiths** [comment]
thanks a lot Amandine. file corrected.

**Comment by danbalogh** [comment]
Yes, `<pb/>` elements should be within `<p>` (or `<ab>` or `<lg>` as the case may be). If the schema does not cry out against it, then it may not cause a processing problem, but I think consistency is best. For consistency, we prescribe that such milestone-like elements must always be within block-level containers, except for lost medial pages (for which there is no text, but the breaks can be encoded) and for the blank first and/or last pages of cp sets.
On line numbering, I think I would prefer consistency within a subcorpus, but there is nothing in our rules forbidding you from using the system associated with your article (EGD §3.2.2: "the preference for either system shall be determined on the level of subcorpora, but may be overridden on a case-by-case basis")

**Comment by arlogriffiths** [comment]
thanks Dan. I have updated the file on this point and will check the other files encoded by Amandine.

I await Ryosuke's response on line-numbering — will be happy to adjust to make for consistency in the Bengal Charters corpus.

**Comment by ryosukefurui** [comment]
Excuse me for a delayed response. The new scheme of line numbering is inconsistent with mine, and also with most of the previous editions of Bengal charters. As we seldom have inscriptions on more than one plate, with possible exception of the Nidhanpur plates of Bhaskaravarman, the continuous line numbering through both sides is rather reasonable and better for line identification. I prefer to leep the present numbering, while being open to keep the cases of the Baigram fragment (and Nidhanpur plates in future) as exceptions, to which we apply page-wise numbering.
